README.md

```diff
@@ -285,6 +285,12 @@ Javadocs and KDocs for the most recent release can be found here: https://uber.g
 
 Snapshots of the development version are available in [Sonatype's snapshots repository][snapshots].
 
+Static code analysis
```

Comment:
just call this `StaticAnalysis`. Also, please move this to above the Download section

Reply:
https://github.com/uber/AutoDispose/pull/156/commits/c51f4db404be41f6df92cd01f5408a3d3f0e8747